apps/etterlatte-pdltjenester/src/main/kotlin/pdl/PdlModell.kt

```diff
@@ -444,4 +445,25 @@ data class PdlGeografiskTilknytningData(
 data class PdlGeografiskTilknytningResponse(
     val data: PdlGeografiskTilknytningData? = null,
     val errors: List<PdlResponseError>? = null
+)
+
+data class PdlHentForelderansvarHistorikkVariables(
+    val ident: String
+) {
+    val historikk: Boolean = true
+}
+
+data class PdlHentForeldreansvarHistorikkRequest(
+    val query: String,
+    val variables: PdlHentForelderansvarHistorikkVariables
+)
+
+data class PdlHentForeldreansvarHistorikkResponse(
+    val data: PdlHistorikkForeldreansvar? = null,
```

Comment:
Betre å defaulte til listOf()?

Reply:
Jeg bare fulgte tidligere konvensjon her, men jeg er enig i at det er en rimeligere default. Jeg ser om jeg endrer det for alle, eller bare denne i første omgang